DefaultTabBar.js

```diff
@@ -45,7 +45,7 @@ const DefaultTabBar = React.createClass({
       accessibilityTraits='button'
       onPress={() => this.props.goToPage(page)}
     >
-      {this.renderTabName(name, page, isTabActive)}
+      {this.props.renderTabName(name, page, isTabActive)}
```

Comment:
btw, looks like we can`t do that:
```
...
renderTabName: this.renderTabName,
...
```

getDefaultProps are called without cuurent context (sort of static method)